src/core/RecyclerListView.tsx

```diff
@@ -151,7 +153,7 @@ export default class RecyclerListView<P extends RecyclerListViewProps, S extends
     private _cachedLayouts?: Layout[];
     private _scrollComponent: BaseScrollComponent | null = null;
 
```

Comment:
mention the reason in comment

Reply:
Comment has been added explaining the reason why baseItemAnimator is used